korim/src/commonMain/kotlin/com/soywiz/korim/format/ASE.kt

```diff
@@ -619,7 +635,7 @@ object ASE : ImageFormatWithContainer("ase") {
                         it.layer, it.slice.slice(sliceFrame, it.slice.name),
                         (if (props.useSlicePosition()) sliceKey.sliceFrame.x else it.targetX) - sliceKey.pivotX,
                         (if (props.useSlicePosition()) sliceKey.sliceFrame.y else it.targetY) - sliceKey.pivotY,
-                        it.main, it.includeInAtlas, null
+                        it.main, it.includeInAtlas
```

Comment:
This null parameter was the property that was removed? It was not used and always null in the end?

Reply:
yea